src/EventJournal.md

```diff
@@ -0,0 +1,75 @@
+## Event journal
+
+The event journal is a distributed data structure that stores the history of mutation actions on data structures such as map or cache. Each action on the map or cache which modifies its contents (such as `put`, `remove` or scheduled tasks which are not triggered by using the public API) will create an event which will be stored in the event journal. The event will store the event type as well as the key, old value and updated value for the entry (when applicable). As a user, you can only append to the journal indirectly by using the map and cache methods or by configuring expiration and eviction. 
```

Comment:
The interaction with expiration and eviction has it's own section now. The replica's won't converge and they will not be eventually consistent since the underlying record store doesn't converge. This is not a fallacy of the event journal, the eviction mechanism should be fixed to have eventually consistent replicas.
I'll think about the graphic.
The event journal is not used from anywhere other than jet code right now. I'm not aware of the use cases, maybe @cangencer can comment on that. If you configure events to be captured in the journal but don't use it (from jet or by using private IMDG API), you would just be wasting heap space, same as with any other structure.